bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -45,11 +45,16 @@
     public static final byte CURRENT_PROTOCOL_VERSION = 2;
```

Comment:
Not changing protocol version? or is it another patch?

Reply:
I think we do not need to bump protocol version as far as new clients which use 'default' LedgerType are able to work with 4.5 bookies.
I will add compat tests, we must be sure that new clients which do not use new ledger types are able to work with old bookies

Reply:
this is V2 protocol, no?